File: Exercicios_Em_Python_CV/desafio_013.py
#Faça um algoritmo que leia o salario de um funcionário
#e mostre o seu novo salário, com 15% de aumento
salario = float(input('Digite o seu novo salário atual R$:'))
novo_salario = salario*1.1 
print("O seu antigo salário era:R${:.2f}".format(salario))
print("O seu novo salário é:R${:.2f}".format(novo_salario))

File: Exercicios_Em_Java_CV/Desafio013.java
package fundamentos;

import java.util.Scanner;

/*
 * Faça um algoritmo que leia o salário de um funcionário e mostre seu novo
 * salário, com 15% de aumento
 */
public class Desafio013 {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		System.out.print("Digite o seu salario:");
		float salario = scanner.nextFloat();
		scanner.nextLine();
		System.out.printf("Salario com reajuste:R$%.2f", salario*1.15);
		scanner.close();
	}

}
